src/main/java/StudentSystem/Model/Course.java

```diff
@@ -0,0 +1,54 @@
+package StudentSystem.Model;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * Created by Fariha Nawaz on 13,Apr,2020.
+ */
+@Entity
+@Table(name= "course")
+public class Course {
+
+    @Id
+    @Column(name = "course_id",nullable = false)
+    private long courseId;
```

Comment:
Ideally, you don't need to add the course prefix to the variables. It is clear enough that the entity is for Course.